app.js

```diff
@@ -39,20 +39,25 @@ app.post("/", async function(request, response) {
 
   // If a new issue was just created
   if (resource === "issue" && action === "created") {
-    // Init or reset queue if empty
-    if (app.queuedUsers.length === 0) {
-      app.queuedUsers = [...app.allUsers];
-    }
-
-    // Assign issue to the next user in the queue and remove user from queue
-    const {id:issueID} = request.body.data.issue;
-    await assignNextUser(issueID);
 
-    if (sentry) {
-      sentry.addBreadcrumb({
-        message: `New issue created. issueID: ${issueID}`,
-        level: sentry.Severity.Info
-      });
+    const {id:newIssueProjectID} = request.body.data.issue.project;
+    // Only continue if this issue is for the project specified in .env
+    if (newIssueProjectID === projectID) {
```

Comment:
@LearningNerd We should probably send a `202` status code when `newIssueProjectID` is different than `projectID`.

Reply:
Hmmm, [`202 Accepted`](https://httpstatuses.com/202) says that we've accepted the event but processing hasn't yet completed. Maybe a `400 Bad Request`?

Reply:
I don't think it makes sense to send a bad request because it's not a bad request. We just aren't doing anything with the request. That's why I think a `202` makes sense.

Reply:
@manuzope But based off our code I see it's mostly used when kicking off async jobs. So I guess that `202` is probably not appropriate.

Reply:
Yea 202s are good when you take an event and put it on a queue. There isn't a great status code for this use case. 200 OK is a good enough option, imo.

Reply:
Thanks for the context on this, I wasn't familiar!